packages/coding-standards/composer.json

```diff
@@ -10,8 +10,8 @@
 	"license": "GPL-2.0-or-later",
 	"require": {
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
```

Comment:
Based on [this commit](https://github.com/PHPCSStandards/PHPCSUtils/commit/fa703bd8f1d824ce5041e10ac220d9cb5faaf691) by the ever-knowing @jrfnl, I think you could have this going down to include `^0.4.1` and `^0.5` as well. Also, judging by that commit, and the release notes at https://github.com/dealerdirect/phpcodesniffer-composer-installer/releases, then you may want to define `^0.6.2` as the minimum of the 0.6.x releases as well.

Reply:
If this standard is intended to be used as a "standard" for other projects and not just and only as a _one-project_ standard, then having a flexible requirement for the Composer plugin is definitely recommended.
If this standard is _only_ intended to be used by _one_ project, then setting it to a "high" version is fine.

Ref: https://github.com/Dealerdirect/phpcodesniffer-composer-installer#requiring-the-plugin-from-within-your-coding-standard

Just FYI: I kind of expect version 0.7 to be released in the semi-near future which should include Composer 2.0 support for the plugin (working on the PR now).

Reply:
That makes a lot of sense. I appreciate the insight. I don't anticipate non-Automattic projects using this ruleset (especially if the hooks docblock sniffs land in WPCS directly), but no need to artificially limit the dependency either.

64f35dd48